.env

```
@@ -0,0 +1,5 @@
+SERVER_PUBLIC_DOMAIN="http://didcomm-mediator.com"
+SERVER_LOCAL_PORT="8080"
+STORAGE_DIRPATH="crates/generic-server/target/storage"
```

Remember that the final application binary will reside in a standalone container, so using this actual storage tree is not the best solution. We could use the same directory as the application, especially since the only file produced is the mediator DID document.
```suggestion
STORAGE_DIRPATH="."
```

---

Since this is something internal to the mediator i preferred removing it completely and external variable and giving it to the application directly